src/main/kotlin/blackjack/domain/Card.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack.domain
+
+import blackjack.enums.Denomination
+import blackjack.enums.Suit
+
+class Card(val denomination: Denomination, val suit: Suit) {
+    val value: Int
+        get() {
+            return denomination.value
+        }
```

Comment:
아래와 같이 코드를 작성해도 동일하게 동작합니다.

denomination의 값은 불변인것 같아보이는데 이렇게 선언해두신 이유가 있을까요?

```suggestion
    val value: Int = denomination.value
```

Reply:
가변 불변 고려 없이 사용했었는데 말씀해주신거 보고 하나 배웠네요